scripts/tests-unit.sh

```diff
@@ -6,16 +6,17 @@ set -o pipefail
 minimum_coverage_percentage=80
 
 this_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd $this_dir/../
+cd /tmp
```

Comment:
Might make sense to use pushd here to make sure we get back to the calling directory

Reply:
good call, I'll do that

Reply:
You forgot to call popd

Reply:
> You forgot to call popd

I think it's unnecessary, no reason to return in the context of the script, but using `pushd` instead of cd makes sense in case we need to return at any point in the evolution of the script. How I understand it, it doesn't matter regardless outside of the context of the script:

```
patrick@patrick sandbox % cat test.sh
#!/bin/bash

pushd /tmp

patrick@patrick sandbox % ./test.sh
/tmp ~/workspace/sandbox
patrick@patrick sandbox % pwd
/Users/patrick/workspace/sandbox
```

So, my shell pwd doesn't matter regardless, just the context of the script.